vtex-apps/my-account
# [B2BORDERS-5] Add settings to show Orders link at the menu

#### What did you change? \*
Added settings configuration to allow hiding Orders link from the menu



#### Why? \*
We've developed a B2B app that handles its own logic for showing Orders History, to avoid any misleading information, we need to hide the default Orders link from the menu



#### How to test it? \*

Check the settings [here](https://demo--b2bstoreqa.myvtex.com/admin/apps/vtex.my-account@1.x/setup/)

![image](https://user-images.githubusercontent.com/24723/139336018-965644ac-7efd-4519-8c01-1d6ee7f940b4.png)
![image](https://user-images.githubusercontent.com/24723/139336073-1a2b7dd7-6792-4fe9-8745-6b6c388cc8dc.png)




#### Related to / Depends on?

[https://github.com/vtex-apps/b2b-orders-history/pull/1](https://github.com/vtex-apps/b2b-orders-history/pull/1)



#### Types of changes \*
